spec/features/guest_registers_for_courses_spec.rb

```diff
@@ -55,7 +55,7 @@
     expect(emiline.course_registrations.first).to be_a(CourseRegistration)
   end
 
-  scenario 'guest registers for four classes' do
+  pending 'guest registers for four classes' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.